<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-6</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>911</width>
      <height>621</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="menu_principal">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>401</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutIngresar_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_enter_user">
           <property name="text">
            <string>Ingrese su usuario:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineedit_enter_user">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="ingresar">
         <property name="text">
          <string>Ingresar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>36</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>PROGRAPOP</string>
      </property>
     </widget>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>label</zorder>
     <zorder>widget</zorder>
     <zorder>widget</zorder>
    </widget>
    <widget class="QWidget" name="menu_salas">
     <widget class="QLabel" name="nombre_usuario_ms">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_salas">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Artistas en la sala</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_salas_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Salas</string>
      </property>
     </widget>
     <widget class="QLabel" name="nombre_usuario_ms_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="puntajes_b">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntajes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sala">
     <widget class="QLabel" name="label_sala">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre Sala</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="boton_op1">
         <property name="text">
          <string>opcion 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boton_op2">
         <property name="text">
          <string>opcion 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boton_op3">
         <property name="text">
          <string>opcion 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boton_op4">
         <property name="text">
          <string>opcion 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="b_salir_sala">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Salir sala</string>
      </property>
     </widget>
     <widget class="QLabel" name="nombre_usuario_s">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="nombre_usuario_s_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntos</string>
      </property>
     </widget>
     <widget class="QLabel" name="tiempo_restante">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tiempo restante: </string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>231</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="nombre_usuario_s_3">
         <property name="text">
          <string>Chat sala</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="chat"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="texto"/>
         </item>
         <item>
          <widget class="QPushButton" name="texto_send">
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>231</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="nombre_usuario_s_4">
         <property name="text">
          <string>Respuestas jugadores</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="respuestas"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="paguina_puntos">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tabla de puntajes</string>
      </property>
     </widget>
     <widget class="QListWidget" name="rank_jugadores">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>331</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ranking jugadores</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>430</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sala con menos aciertos:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sala con mas aciertos:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="volver">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
     <widget class="QLabel" name="rank_aciertos">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>180</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sala con mas aciertos:</string>
      </property>
     </widget>
     <widget class="QLabel" name="rank_aciertosmalos">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>450</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sala con menos aciertos:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
